script.js

```diff
@@ -15,4 +15,137 @@ window.onload = () => {
   menu.addEventListener('click', () => {
     popUp.classList.add('hidden');
   });
-};
\ No newline at end of file
+
+  const projectsData = [
+    {
+      title: 'Tonic',
+      subtitle: 'CANOPY',
+      techStack: 'Back End Dev',
+      year: 2015,
+      src: './images/second-section/project-01.svg',
+      highlight: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+      lang: ['html', 'css', 'javascript'],
+      live: 'https://kanzatahreem.github.io/Portfolio-Website/',
+      source: 'https://github.com/KanzaTahreem/Portfolio-Website',
+    },
+    {
+      title: 'Multi Post Stories',
+      subtitle: 'CANOPY',
+      techStack: 'Back End Dev',
+      year: 2015,
+      src: './images/second-section/project-02.svg',
```

Comment:
- [ ] please kindly make the mobile version responsive when switching to deskstop version
![Screenshot (435)](https://user-images.githubusercontent.com/65298048/188756107-2c3ae21c-fe14-4052-81b8-26c40a9e2012.png)